### PR TITLE
Adding actix-web example

### PR DESCRIPTION
**Pull request by knabben**
Initial actix-web middleware and parent span testing.

image

Missing metrics and move init_trace to a reusable module.

Run with: cargo run --example actix
Test with: curl localhost:8088/

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -20,6 +20,9 @@ rand = { version = "0.7.2", optional = true }
 hyper = "0.12.0"
 opentelemetry-jaeger = { path = "opentelemetry-jaeger", version = "0.1.0" }
 thrift = "0.13.0"
+futures = "0.1.25"
```
Comment (iredelmeier) [inline Cargo.toml]: Mind moving these deps so that they're only added for the example? (I can edit this PR or add a PR to your branch if you want.)

Comment (knabben) [reply Cargo.toml]: please feel free to add commits on this one.

**Comment by jtescher** [comment]
Should this include the [server semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#http-server-semantic-conventions)? Have an example at https://github.com/OutThereLabs/actix-web-opentelemetry/ which includes many of them. Does add some complexity though, could be fine to have this as a simpler example as well to show usage more than full spec.

File: Cargo.toml (hunk 1 above)
Comment (iredelmeier) [reply Cargo.toml]: Thanks!

**Comment by iredelmeier** [comment]
@jtescher I think that would be a good addition but isn't necessary, given that this is an example (as opposed to a plugin that someone would import directly).

**Pull request merged**